src/main/java/BrainRot/exceptions/UnknownLoadingError.java

```diff
@@ -0,0 +1,9 @@
+package BrainRot.exceptions;
+
+import java.io.IOException;
+
+public class UnknownLoadingError extends IOException{
```

Comment:
Using the Exception or IOException suffix instead of Error may be more appropriate.